langmeta-sqlite/src/main/resources/semanticdb.ddl

```diff
@@ -0,0 +1,49 @@
+create table document(
+  id int,
+  filename text,
```

Comment:
Moreover, speaking of the lowlevel layer of the protobuf schema (`org.langmeta.internal.semanticdb.schema`). What would it take to make it as attractive to users as `org.langmeta.semanticdb`? 

I think it's a shame that only Scala people can enjoy some of the conveniences that we provide in the highlevel API. Maybe we could push them to the lowlevel API, too?

Reply:
/cc @olafurpg

Reply:
I think we could just combine the schemas easily enough and retain the performance of internal.semanticdb.schema.  Should just be adding a bunch of extension methods to the low level version right?